docs/content/getting-started/perform-list-users.mdx

```diff
@@ -61,7 +61,7 @@ This section will illustrate how to perform a <ProductConcept section="what-is-a
 
 </TabItem>
 
-{/* <TabItem value={SupportedLanguage.PYTHON_SDK} label={languageLabelMap.get(SupportedLanguage.PYTHON_SDK)}>
+{ /* <TabItem value={SupportedLanguage.PYTHON_SDK} label={languageLabelMap.get(SupportedLanguage.PYTHON_SDK)}>
```

Comment:
(Not related to this PR: can you please fix the typo in line 31?

> `with a given <ProductConcept section="what-is-an-object" linkName="objects" />.`

should be 

> `with a given <ProductConcept section="what-is-an-object" linkName="object" />.`